### PR TITLE
Imie, nazwisko i avatar uzytkownika

### PR DESCRIPTION
**Pull request by wiaterb**

### CODE REVIEW

**Comment by wiaterb** [comment]
https://kindergarten.stxnext.pl/browse/WDP-42

File: runtime/data/test_users.xml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<intranet>
+    <server>
+        <host>intranet.stxnext.pl</host>
+        <port>443</port>
+        <protocol>https</protocol>
+    </server>
+    <users>
```
Comment (dpolcyn) [inline runtime/data/test_users.xml]: I think you should add one more user

File: src/presence_analyzer/static/js/utils.js [hunk 2]
```diff
@@ -17,3 +17,23 @@ function parseInterval(value) {
...
+            var avatar = $('#avatar');
+            avatar.hide();
+            var real_name = $('#real_name_info');
+            real_name.hide()
+            var selected_user = $("#user_id").val();
+            $.getJSON("/api/v1/users/"+selected_user, function(result) {
+                if(result.avatar){
+                    avatar.empty().prepend("<img src='" + result.avatar + "'/>").show();
+                }
+                if(!result.real_name){
```
Comment (dpolcyn) [inline src/presence_analyzer/static/js/utils.js]: If you have result.avatar, then you should have result.real_name too. Only one case is that you have avatar, name but you don't have data for specified user.

File: src/presence_analyzer/static/js/utils.js [hunk 3]
```diff
@@ -17,3 +17,23 @@ function parseInterval(value) {
...
+                    real_name.empty().prepend("There is no user\'s real name").show();
```
Comment (dpolcyn) [inline src/presence_analyzer/static/js/utils.js]: users

File: src/presence_analyzer/tests.py [hunk 4]
```diff
@@ -70,7 +77,27 @@ def test_api_users(self):
         self.assertEqual(resp.content_type, 'application/json')
         data = json.loads(resp.data)
         self.assertEqual(len(data), 2)
-        self.assertDictEqual(data[0], {u'user_id': 10, u'name': u'User 10'})
+        self.assertDictEqual(data[0], {u'user_id': 10, u'name': u'Jan P.'})
```
Comment (dpolcyn) [inline src/presence_analyzer/tests.py]: from __future__ import unicode_literals
then you don't have to use 'u

File: src/presence_analyzer/tests.py [hunk 5]
```diff
@@ -70,7 +77,27 @@ def test_api_users(self):
...
+
+    def test_users_data_api_view(self):
+        """
+        Test user data view.
+        """
+        resp = self.client.get('/api/v1/users/10')
+        self.assertEqual(resp.status_code, 200)
+        self.assertEqual(resp.content_type, 'application/json')
+        data = json.loads(resp.data)
+        self.assertEqual(
```
Comment (dpolcyn) [inline src/presence_analyzer/tests.py]: sample_data = {.....}
then assert dict

File: src/presence_analyzer/utils.py [hunk 6]
```diff
@@ -71,6 +72,46 @@ def get_data():
...
+    name_reader = etree.parse(app.config['DATA_XML'])
+    avatar_base_url = '{0}://{1}'.format(
+        name_reader.find('server').find('protocol').text,
+        name_reader.find('server').find('host').text
+    )
+    for user in name_reader.find('users').findall('user'):
+        user_id = int(user.attrib['id'])
+
+        try:
+            data[user_id]['avatar'] =\
```
Comment (dpolcyn) [inline src/presence_analyzer/utils.py]: .... = '{0}{1}.format(
.....
)

**Pull request merged**